_includes/nav.html

```diff
@@ -39,7 +39,7 @@
 
         <ul class="social">
             <li role="presentation" class="social__twitter">
-                <a href="https://twitter.com/intent/tweet" class="twitter-mention-button" data-text="Support #Democracy4All this #Election2016. Sign up on @CarpoolVote: Linking drivers with voters needing a ride" data-url="http://carpoolvote.com" data-related="" data-show-count="false">Tweet to help others find out about this!</a>
+                <a href="https://twitter.com/intent/tweet" class="twitter-mention-button" data-text="Support #VotingRights in 2017 #USelections. Sign up via @carpoolvote: Linking drivers with voters needing a ride. carpoolvote.com" data-url="http://carpoolvote.com" data-related="" data-show-count="false">Tweet to help others find out about this!</a>
```

Comment:
There's no need to add the URL to the tweet text, as it's already included by the `data-url` attribute. It also makes the tweet too long:
![image](https://cloud.githubusercontent.com/assets/1155816/26444955/3c6bbba0-4136-11e7-8459-253cbba62c6d.png)
Please change to the following:
```
data-text="Support #VotingRights in 2017 #USelections. Sign up via @carpoolvote: Linking drivers with voters needing a ride."
```